tests/ios/RealmJSCoreTests.m

```diff
@@ -48,6 +48,8 @@ + (XCTestSuite *)defaultTestSuite {
         context[@"Promise"] = promiseModule[@"Promise"];
     }
 
+    context[@"global"] = [JSValue valueWithNewObjectInContext:context];
```

Comment:
Wonder why do we need this. I am afraid it might hurt us if 'global' should not be present in that context.

Reply:
The tests expect to be able to pass state between modules by assigning to properties of a variable named `global`, and that doesn't actually exist in RN (but does in everything else we run tests in).

Reply:
yes my concern is we may introduce bugs where RN (non test) code referring to 'global' might succeed in our tests but can fail elsewhere. Maybe we need to fix our tests instead. But  probably should create an issue for that

Reply:
That's a valid concern, but I think waiting until after the release to deal with that sounds like the right choice.